packages/node_modules/@webex/plugin-memberships/src/memberships.js

```diff
@@ -41,8 +41,44 @@ const debug = require('debug')('memberships');
  */
 const Memberships = WebexPlugin.extend({
   /**
-   * Connect to the web socket to listen to incoming messages.
+   * Register to listen for incoming membership events
+   * This is an alternate approach to registering for membership webhooks.
+   * The events passed to any registered handlers will be similar to the webhook JSON,
+   * but will omit webhook specific fields such as name, secret, url, etc.
+   * See the <a href="https://js.samples.s4d.io/browser-socket/">Sample App</a>
+   * for more details.
+   * @instance
+   * @memberof Memberships
    * @returns {Promise}
+   * @example
+   * webex.memberships.listen()
+   *   .then(() => {
+   *     console.log('listening to membership events');
```

Comment:
We should only console.log in samples. Let's use the logger here so output can be controlled.